.github/workflows/preview.yml

```diff
@@ -27,10 +27,6 @@ jobs:
       - run: |
           git clone --depth 1 --single-branch --branch ${{ inputs.branch }} https://github.com/$GITHUB_REPOSITORY
           cd apollo-kotlin
-          sed -i .old 's/VERSION_NAME=\([\.0-9]*\).*/VERSION_NAME=\1-${{ inputs.prerelease }}/g' gradle.properties
-          export GOOGLE_APPLICATION_CREDENTIALS=google-services.json
-          echo ${GOOGLE_SERVICES_JSON} > $GOOGLE_APPLICATION_CREDENTIALS
-          ./gradlew publishAllPublicationsToApolloPreviewsRepository
+          LIBRARIAN_NIGHTLY ./gradlew librarianPublishToGcs
```

Comment:
Should this be `LIBRARIAN_NIGHTLY=true`? I think the nightlies have been [failing](https://github.com/apollographql/apollo-kotlin/actions/runs/16433891900/job/46440158912#step:4:20) on main actually.

Reply:
Good catche 👍